### PR TITLE
SUBS-587 ReStyle BT DropIn UI

### PR DESCRIPTION
**Pull request by eddieferrer**
* Restyled DropIn UI according to new design specs

### CODE REVIEW

**Comment by eddieferrer** [comment]
This is approved by design, mostly. There might be a small follow-up PR to adjust the CC icons.

File: src/assets/scss/_settings.scss [hunk 1]
```diff
@@ -102,6 +102,8 @@ $dark-gray: #666;
 $charcoal: #484848;
 $dark-charcoal: #333;
 $black: #000;
+$magnemite: #757575;
```
Comment (eddieferrer) [inline src/assets/scss/_settings.scss:105]: 2 new colors

File: src/components/Payment/BraintreeDropInInterface.vue [hunk 2]
```diff
@@ -202,25 +202,29 @@ export default {
...
+$form-border-radius: rem-calc(4);
 $active-border-color: $input-checked-color;
 $icon-background-color: $input-checked-color;
 $border-width: 1px;
 
-#dropin-container {
-	::v-deep [data-braintree-id="choose-a-way-to-pay"] {
-		font-size: $small-text-font-size;
-	}
+// !TODO Change uses of $umbreon here for the functional name.
```
Comment (eddieferrer) [inline src/components/Payment/BraintreeDropInInterface.vue:210]: Not sure what the functional name for $umbreon will be

Comment (emuvente) [reply src/components/Payment/BraintreeDropInInterface.vue:210]: I believe it's meant to be $kiva-glyph-primary-black, which is replacing $kiva-text-dark

File: src/assets/scss/global/drop-in.scss [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+	line-height: rem-calc(14);
+}
+
+.dropin-payment-holder {
+	width: 100%;
+	padding: 0 1.25rem;
+	text-align: left;
+
+	@include breakpoint(medium) {
+		max-width: 400px;
```
Comment (emuvente) [inline src/assets/scss/global/drop-in.scss:18]: minor nitpick: rem-calc(400)

**Review by emuvente — APPROVED** [review]
new colors! 👍

**Review by mcstover — APPROVED** [review]

**Review by mcstover — APPROVED** [review]

**Pull request merged**